### DISCUSSION TITLE
Performance of beatmap listing overlay is poor

### DISCUSSION THREAD

**Discussion by maximium91** [ppy/osu#21005] [Q&A] [answered]
### Type

Performance

### Bug description

Pulling up the beatmap listing overlay drags framerate down quite a lot, scrolling makes it much worse.

### Screenshots or videos

https://user-images.githubusercontent.com/107336266/198792823-c3a7ec11-cddc-4a03-b1ad-128c4020593e.mp4

### Version

2022.1022.0

### Logs

[performance.log](https://github.com/ppy/osu/files/9892708/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9892709/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9892710/updater.log)
[input.log](https://github.com/ppy/osu/files/9892711/input.log)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Overlays such as beatmap listing, especially when shown above song select, result in a lot of elements being drawn on screen. We'll continue to optimise such scenarios as we have time.

As a temporary solution, you can try using it from the main menu. It should be a bit more usable.
